.codesandbox/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
You can ignore this, this is just some settings so you can use [codesandbox.com](https://codesandbox.io/p/github/michalvarys/celestial-academy) as a playground to see the changes real-time without having to set up your PC. You can use it to build your website too.